tests/experimental/pydantic/test_fields.py

```diff
@@ -176,7 +188,9 @@ class UserType:
         pydantic.PaymentCardNumber,
         pydantic.ByteSize,
         # pydantic.JsonWrapper,
-    ],
+    ]
+    if IS_PYDANTIC_V1
+    else [],
```

Comment:
need to do this trick otherwise it will still getattr on the pydantic namespace and explode